src/chainparams.cpp

```diff
@@ -435,6 +435,7 @@ class CCustomParams : public CRegTestParams {
         // All non-zero coinbase outputs must go to this scriptPubKey
         std::vector<unsigned char> man_bytes = ParseHex(gArgs.GetArg("-con_mandatorycoinbase", ""));
         consensus.mandatory_coinbase_destination = CScript(man_bytes.begin(), man_bytes.end()); // Blank script allows any coinbase destination
+        initialFreeCoins = gArgs.GetArg("-initialfreecoins", 0);
```

Comment:
use `DEFAULT_INITIAL_FREE_COINS`

Reply:
Would cause circular dependencies... Just used 0

Reply:
well that constant doesn't appear to be used then

Reply:
@instagibbs It would seem from surrounding code that the convention is to just specify instead of making a new constant. So I've removed it.